internal/io/csv.go

```diff
@@ -98,6 +101,40 @@ func ReadCSV(reader io.Reader, conf CSVConfig) (map[string]interface{}, []string
 		}
 	}
 
+	if conf.MissingColumnNameAlias != "" {
+		for i, name := range headers {
+			if name == "" {
+				headers[i] = conf.MissingColumnNameAlias
+			}
+		}
+	}
+
+	if conf.AddColumnIndexToDuplicateColumns {
+		headersMap := make(map[string]int)
+		// Loop through all columsn and find the duplicates.
+		for i, h := range headers {
+			headersMap[h] = i
+		}
+		// For go through all if there is duplicate down the line add index
+		for i, h := range headers {
+			index, ok := headersMap[h]
+			if ok && i != index {
+				counter := 0
+				for {
+					_, ok = headersMap[headers[i]+fmt.Sprint(counter)]
```

Comment:
Inline and assign `_, ok =` in the if-expression to reduce the scope of the variable and avoid overwriting the existing `ok` variable?